app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -1870,6 +1918,8 @@ class RenderMarkup extends React.Component {
           metric={parent.state.treeMetric}
           nameType={parent.state.name_type}
           onNodeTextClicked={this._nodeTextClicked}
+          backgroundId={parent.state.backgroundId}
```

Comment:
Any reason not to pass an object with `backgroundInfo = {name:..., value:...}` here?

Reply:
sounds good